tests/migration_test.go

```diff
@@ -542,6 +542,37 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 				tests.WaitForVirtualMachineToDisappearWithTimeout(vmi, 240)
 
 			})
+			It("should migrate to a persistent (non-transient) libvirt domain.", func() {
+				vmi := tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskCirros))
+				tests.AddUserData(vmi, "cloud-init", "#!/bin/bash\necho 'hello'\n")
+
+				By("Starting the VirtualMachineInstance")
+				vmi = runVMIAndExpectLaunch(vmi, 240)
+
+				By("Checking that the VirtualMachineInstance console has expected output")
+				Expect(libnet.WithIPv6(console.LoginToCirros)(vmi)).To(Succeed())
+
+				// execute a migration, wait for finalized state
+				By(fmt.Sprintf("Starting the Migration"))
+				migration := tests.NewRandomMigration(vmi.Name, vmi.Namespace)
+				migrationUID := tests.RunMigrationAndExpectCompletion(virtClient, migration, migrationWaitTime)
+
+				// check VMI, confirm migration state
+				tests.ConfirmVMIPostMigration(virtClient, vmi, migrationUID)
+
+				// make sure no transient libvirt domain is running
+				transient, err := tests.FindTransientDomains(virtClient, vmi)
```

Comment:
Perhaps we should do the opposite and look for persistent domains to find ours in it.
I mean s/FindTransientDomains/FindPersistentDomains
Otherwise, it opens the door for the test to still pass when the domain has actually crashed...

Reply:
what do you think?

Reply:
Yeah I went back-and-forth about it, but your're making a good point about the domain crashing.
I can write a function that ensures only one domain is running and that it is persistent. The hardest part will be to name that function!

Reply:
How about `func CountTransientAndPersistentDomains(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance) (int, int, error)`?
That would be versatile enough to be reusable. I'd `Expect()` `0, 1, nil`.

Reply:
Sure, that will work. But for me, it would be enough to find the destination domain as persistent.
Otherwise, for completeness, I guess CountTransientAndPersistentDomains is good as well :)

Reply:
> I can write a function that ensures only one domain is running and that it is persistent. The hardest part will be to name that function!

ha, it's always the case for me too :) 
I think something like `ListPersistentDomains` and find the destination one in that list, would be good from my point of view.

Reply:
In the end, a version of `IsPersistent` is what made the most sense... Force-pushed.